### DISCUSSION TITLE
关于视频字幕问题

### DISCUSSION THREAD

**Discussion by igoguojia** [AlistGo/alist#561] [Q&A] [answered]
是不支持srt格式的字幕吗？播放时会乱码
![](https://p.sda1.dev/5/e76deba2f5f7f7ad461c208c71e0e204/QQ截图20220218194740.png)![](https://p.sda1.dev/5/a7232a723e65f51c92fc0c5bf48118a3/QQ截图20220218194754.png)

**Comment by xhofe** [comment, marked as answer]
Go: https://github.com/zhw2590582/ArtPlayer
